### PR TITLE
Fuzz test permissions manager module

### PR DESCRIPTION
**Pull request by comeonbuddy**
### Please check if the PR fulfills these requirements
- [x] The commit message follows our Submission guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce? 
(Bug fix, feature, docs update, ...)

### What is the current behavior? 
(You can also link to an open issue here)

### What is the new behavior?
(Define and describe any new functionality. Clarify if this is a feature change)

### Does this PR introduce a breaking change?
(What changes might users need to make in their application due to this PR?)

### CODE REVIEW

**Comment by satyamakgec** [comment]
@comeonbuddy when this will be complete?

**Comment by satyamakgec** [comment]
@comeonbuddy could you please remove all the commented part, It is not the best practices to keep the unused code in the repo. And please put a comment on the test cases about the logic so it will be easy to review the code.

File: test/z_general_permission_manager_fuzzer.js [hunk 1]
```diff
@@ -0,0 +1,1263 @@
...
+} from "./helpers/createInstances";
+import { encodeModuleCall } from "./helpers/encodeCall";
+
+const SecurityToken = artifacts.require('./SecurityToken.sol');
+const GeneralTransferManager = artifacts.require('./GeneralTransferManager');
+const GeneralPermissionManager = artifacts.require('./GeneralPermissionManager');
+const CountTransferManager = artifacts.require("./CountTransferManager");
+const VolumeRestrictionTransferManager = artifacts.require('./LockupVolumeRestrictionTM');
+const PercentageTransferManager = artifacts.require('./PercentageTransferManager');
+const SingleTradeVolumeRestrictionManager = artifacts.require('./SingleTradeVolumeRestrictionTM');
```
Comment (satyamakgec) [inline test/z_general_permission_manager_fuzzer.js]: Remove this, we no longer supporting the single-tradeVRTM. Also, remove from the createInstances.js if it present

File: test/z_general_permission_manager_fuzzer.js [hunk 2]
```diff
@@ -0,0 +1,1263 @@
...
+	    // Deploy Modules
+        [I_CountTransferManagerFactory] = await deployCountTMAndVerifyed(account_polymath, I_MRProxied, I_PolyToken.address, 0);
+
+        [I_VolumeRestrictionTransferManagerFactory] = await deployLockupVolumeRTMAndVerified(account_polymath, I_MRProxied, I_PolyToken.address, 0);
+
+        [I_PercentageTransferManagerFactory] = await deployPercentageTMAndVerified(account_polymath, I_MRProxied, I_PolyToken.address, 0);
+
+        [I_SingleTradeVolumeRestrictionManagerFactory] = await deploySingleTradeVolumeRMAndVerified(account_polymath, I_MRProxied, I_PolyToken.address, 0);
+        [P_SingleTradeVolumeRestrictionManagerFactory] = await deploySingleTradeVolumeRMAndVerified(account_polymath, I_MRProxied, I_PolyToken.address, web3.utils.toWei("500"));
+
```
Comment (satyamakgec) [inline test/z_general_permission_manager_fuzzer.js]: Remove it

**Review by satyamakgec — DISMISSED** [review]
You have to remove the STVRTM related code as we are no longer using that contract

File: test/z_general_permission_manager_fuzzer.js (hunk 1 above)
Comment (adamdossa) [reply test/z_general_permission_manager_fuzzer.js]: @comeonbuddy Please can you take a look at this.

File: test/z_general_permission_manager_fuzzer.js (hunk 2 above)
Comment (adamdossa) [reply test/z_general_permission_manager_fuzzer.js]: @comeonbuddy Please can you take a look at this.

File: test/z_general_permission_manager_fuzzer.js (hunk 1 above)
Comment (comeonbuddy) [reply test/z_general_permission_manager_fuzzer.js]: done!

File: test/z_general_permission_manager_fuzzer.js (hunk 2 above)
Comment (comeonbuddy) [reply test/z_general_permission_manager_fuzzer.js]: done

**Comment by maxsam4** [comment]
@comeonbuddy Please remove all the unused commented code. 
I'll complete the review tomorrow.

**Review by adamdossa — APPROVED** [review]

**Pull request merged**